src/frontend/screens/Onboarding/welcome/index.module.scss

```diff
@@ -32,8 +39,38 @@
 .welcomeBodyTextContainer a:hover {
   color: var(--color-primary-300);
   transition: 0.2s;
+  font-size: var(--font-size-md);
```

Comment:
Do we need this? It seems odd to me that we would set the font size instead of using a typography style, especially on hover

Reply:
This was already in the file, I only changed it from lg to md;